rake-tasks/crazy_fun/mappings/ruby.rb

```diff
@@ -198,8 +196,26 @@ def define_gem_install_task(dir, _args)
       desc 'Install gem dependencies for the current Ruby'
       task "//#{dir}:bundle" do
         ENV['BUNDLE_GEMFILE'] = 'rb/Gemfile'
-        sh "gem install bundler"
-        sh "bundle", "install"
+
+        gem_path = "#{Dir.pwd}/build/third_party/rb"
+        bin_path = "#{gem_path}/bin"
+        ENV["GEM_PATH"] = gem_path
+        ENV["PATH"] = [bin_path, ENV["PATH"]].join(":")
+
+        gems = `gem list`.split("\n")
```

Comment:
When would this be empty?

Reply:
When you don't have bundler installed ;)